firmware/console/binary/output_channels.txt

```diff
@@ -74,7 +74,6 @@ uint16_t rpmAcceleration;dRPM;"RPM/s",1, 0, 0, 5, 2
 	uint16_t autoscale chargeAirMass;@@GAUGE_NAME_AIR_MASS@@;"g",{1/1000}, 0, 0, 0, 3
 	uint16_t autoscale crankingFuelMs;crankingFuelMs\nairmass in mg, 0-65 grams;"ms",{1/@@PACK_MULT_FUEL_MASS@@}, 0, 0, 0, 2
 
-	uint16_t autoscale currentTargetAfr;@@GAUGE_NAME_TARGET_AFR@@;"ratio",{1/@@PACK_MULT_AFR@@}, 0, 0, 0, 2
```

Comment:
this is not part of "constexpr fuel computer"

Reply:
That is not in the total diff, are you looking at a single commit?

Reply:
yes, i look at single commits to divide and conquer

Reply:
that commit is from a previous PR, I branched before it merged.